fetch-polyfill.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+// Polyfill from https://github.com/github/fetch/blob/v1.1.1/fetch.js#L8-L21
```

Comment:
Let's get some permanent links to all linked files.

While viewing the file in github, you can press `y` to transform the URL into a permanent form.